index.js

```diff
@@ -44,15 +41,15 @@ var cssLintPlugin = function(options) {
   var rcLoader = new RcLoader('.csslintrc', options, { loader: 'async' });
 
   // Build a list of all available rules
-  csslint.getRules().forEach(function(rule) {
+  csslint.getRules().forEach(function (rule) {
```

Comment:
Spaces before braces doesn't match my personal coding style, can we change that?